### PR TITLE
use local conf file by default, abstract EDITOR choice

### PR DESCRIPTION
**Pull request by notklaatu**
I added a conf file allowing users to override default DOCROOT and other vars.

Also added a sample conf file.

Also abstracted EDITOR. Should respect user's env, or the user can choose one in their conf file. I have EDITOR set in so many places at this point, it's hard for me to test, but it should work...

### CODE REVIEW

File: dot-local/share/nepenthes.conf-example [hunk 1]
```diff
@@ -0,0 +1,5 @@
+DOCROOT="$HOME/notes"
+TODAY=`date "+%Y%m%d"`
+END=`date -d "+7 days" "+%Y%m%d"`
+AUTOCREATE=1 #0 for false
+EDITOR=emacs
```
Comment (natzert) [inline dot-local/share/nepenthes.conf-example]: No, you monster. haha

Comment (natzert) [inline dot-local/share/nepenthes.conf-example]: Maybe should just change `bin/nepenthes` to use $EDITOR

**Comment by notklaatu** [comment]
updated. tested and confirmed: setting EDITOR var in .bashrc or .cshrc is respected with my changes to the nepenthes script, so it's unnecessary to set it in conf.

**Comment by natzert** [comment]
:+1:

**Pull request merged**